### ISSUE TITLE
HTTP API Delete Bug

### ISSUE THREAD

**Issue by karrilamb** [OpenTSDB/opentsdb#1028]
I am using the http api to delete a metric, but after the deletion, the metric still shows in queries. In addition, after deleting the metric, the metric's datapoints are empty and the last timestamp has been set to zero. 

I am running the delete from a  Node.js delete request with a url like the following:
`https://tsdb-server:9443/api/query?start=1y-ago&,m=sum:testMetric`

I get a 200 response even though the metric still appears in subsequent queries.

**Comment by manolama** [comment]
Hi, are you calling that url with the ``DELETE`` verb? And have you configured ``tsd.http.query.allow_delete=true`` in your TSD conf?

**Comment by karrilamb** [comment]
@manolama Yes, calling with DELETE verb. And yes, configured tsd.http.query.allow_delete=true
